server/src/main/resources/static/old-console/js/rules-0.0.2.js

```diff
@@ -151,6 +155,10 @@ function makeDropByPeriod(rule) {
   return "<span class='rule_label'>period</span><input type='text' name='period' " + "value='" + rule.period + "'/>";
 }
 
+function makeDropBeforeByPeriod(rule) {
+  return "<span class='rule_label'>period</span><input type='text' name='period' " + "value='" + rule.period + "'/>";
```

Comment:
would it make sense to put something more descriptive so it is clear this `period` is different than the one above it?

Reply:
Ok. Sounds good.